gtk/OptionsDialog.cc

```diff
@@ -335,7 +335,10 @@ void TorrentFileChooserDialog::onOpenDialogResponse(int response, Glib::RefPtr<S
     if (response == TR_GTK_RESPONSE_TYPE(ACCEPT))
     {
         /* remember this folder the next time we use this dialog */
-        gtr_pref_string_set(TR_KEY_open_dialog_dir, IF_GTKMM4(get_current_folder, get_current_folder_file)()->get_path());
+        if (auto const folder = IF_GTKMM4(get_current_folder, get_current_folder_file)())
```

Comment:
I'm having a hard time understanding how GTK works here (or rather why it works as it does) :-\ It's a good change in that it will fix the reported crash, but it will then silently disregard the last folder and I'm not even sure if it'll ever remember it. Maybe we should use `files.front()->get_parent()->get_path()` instead?..

```suggestion
        if (auto const folder = IF_GTKMM4(get_current_folder, get_current_folder_file)(); folder != nullptr)
```

Reply:
Added this as fallback as you suggested